omod/src/main/resources/moduleApplicationContext.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you let me know why you created this file?

Reply:
@wikumChamith Thanks for the point out by removing the duplicate and running this command `ls api/target/classes/moduleApplicationContext.xml` gives this output `api/target/classes/moduleApplicationContext.xml`confirming Maven has copied `moduleApplicationContext.xml` into target/classes for the test runtime.